mne_bids/write.py

```diff
@@ -386,16 +438,31 @@ def _coordsystem_json(raw, unit, orient, manufacturer, fname,
             coords['coil%d' % ident] = hpi[ident]['r'].tolist()
 
     coord_frame = set([dig[ii]['coord_frame'] for ii in range(len(dig))])
-    if len(coord_frame) > 1:
-        err = 'All HPI and Fiducials must be in the same coordinate frame.'
-        raise ValueError(err)
+    if len(coord_frame) > 1:  # noqa E501
```

Comment:
why ignore E501?

Reply:
Wasn't sure how to ignore this line from being included in the codecov, but you're right this is the run `noqa`.